pkg/migrations/internal/seq_num.go

```diff
@@ -4,7 +4,7 @@ var (
 	// CurrentDBVersionSeqNum is the current DB version number.
 	// This must be incremented every time we write a migration.
 	// It is a shared constant between central and the migrator binary.
-	CurrentDBVersionSeqNum = 172
+	CurrentDBVersionSeqNum = 173
```

Comment:
173 was just pushed.

Reply:
Re-aligned migration versions, leaving space for https://github.com/stackrox/stackrox/pull/5142